protocol/activity_center.go

```diff
@@ -100,8 +102,12 @@ type ActivityCenterNotification struct {
 	TokenData                 *ActivityTokenData             `json:"tokenData"`
 	//Used for synchronization. Each update should increment the UpdatedAt.
 	//The value should represent the time when the update occurred.
-	UpdatedAt     uint64            `json:"updatedAt"`
-	AlbumMessages []*common.Message `json:"albumMessages"`
+	UpdatedAt                  uint64            `json:"updatedAt"`
+	AlbumMessages              []*common.Message `json:"albumMessages"`
+	WalletProviderSessionTopic string            `json:"walletProviderSessionTopic"`
```

Comment:
It would be good to add `omitempty` in these new fields, otherwise many types of notifications will be sent with extra keys containing only zero values, which in turn will increase the cost for the Clojure layer to transform from JS objects.

Reply:
thanks, fixed